app/helpers/exp_atom_helper.rb

```diff
@@ -17,4 +17,45 @@ module ExpAtomHelper
   VM_EXP_TYPES = [
     [N_("Registry"), "regkey"]
   ].freeze
+
+  def self.expression_types_for(model, expkey, edit_expkey)
+    expression_types = EXP_TYPES.map { |x| [_(x[0]), x[1]] }
+    opts = []
+    unless model == "_display_filter_"
+      # Not valid for secondary display filter
+      unless MiqExpression.miq_adv_search_lists(model, :exp_available_counts).empty?
+        expression_types -= [[_(EXP_COUNT_TYPE[0]), EXP_COUNT_TYPE[1]]]
```

Comment:
@mzazrivec do we need `_(` here ? (we have already  `N(` for `EXP_COUNT_TYPE` constant) thanks!